runtime/modules/json/core/src/test/java/io/atlasmap/json/core/JsonFieldReaderTest.java

```diff
@@ -748,4 +752,821 @@ private void resetField(JsonField field) {
         field.setValue(null);
         field.setFieldType(null);
     }
+
+    @Test
+    public void testJsonFieldDoubleMax() throws Exception {
+        String filePath = "src" + File.separator + "test" + File.separator + "resources"
```

Comment:
Could we make some private/protected methods for repetitious code? There seems to a few standard code patterns in use in the file.

Reply:
Done. extracted private method for repetitive codes. Please review. Thanks.